docs/design/proposals/direct-blk-device-assignment.md

```diff
@@ -0,0 +1,264 @@
+# Background
+Today, there exists a few gaps between Container Storage Interface (CSI) and virtual-machine(VM) based runtimes like Kata Containers 
+that prevent them from working together smoothly.
+
+First, it’s cumbersome to use a persistent volume (PV) with Kata Containers. Today, for a PV with Filesystem volume mode, `virtiofs`
```

Comment:
You can say "Virtio-fs" as the spell check dictionary knows about that (https://github.com/kata-containers/tests/blob/main/cmd/check-spelling/data/acronyms.txt).